Proposal.md

```diff
@@ -0,0 +1,65 @@
+# Project Proposal: Predicting the Court Minutes of Players in NBA
+
+<br>
+
+## Motivation and Purpose
+
+<br>
+
+The motivation behind this project is to help make decisions when placing bets on NBA matches. In other words, we are applying "machine-learning" algorithms to help predict the outcomes. However, since match results are decided by many factors, we will only be focusing on predicting the number of minutes that each player will play on the respective teams. It has been shown that the number of minutes played correlates well with the number of points scored, and thus the total score of a team can be estimated from such. 
```

Comment:
I don't think that we can use this so much for betting on matches, as minutes are more related to the odds, and not the other way around. I would change betting to refer to daily fantasy sports, whereby player production is measured, and minutes played are highly correlated with the players output resulting in higher fantasy points. The daily fantasy sports market represents billions of dollars in investment, and is expected to continue growing with a rule change allowing Sports Betting at the state level by the US Supreme Court in 2018.